lib/Exposed.ml

```diff
@@ -73,7 +74,9 @@ module Right = struct
         list ~elt:constructor_declaration cdecls
 
   let type_extension = function
-    | {ptyext_attributes= _ :: _; _} -> false
+    | {ptyext_attributes; _} when Ast.Ext_attrs.has_attrs ptyext_attributes
```

Comment:
Same bug as above, this should check only for `attrs_after`. Two other cases below.

I wonder if that causes the small regressions found by test-branch.

Reply:
This bug happens two more times in the same module.

Reply:
This is fixed